configuration/appframework/registrationapp/editPatientBillingInfo_extension.json

```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "id": "registrationapp.editPatientBillingInfo",
+        "extensionPointId": "patientHeader.editPatientBillingInfo",
+        "type": "link",
+        "label": "general.edit",
+        "url": "registrationapp/editSection.page?patientId={{patient.patientId}}&sectionId=billingInfo&appId=referenceapplication.registrationapp.registerPatient"
```

Comment:
Makes sense, though we don't actually have a billingInfo section in the registration app yet, do we?

Reply:
We don't for now, just did it when I was exploring extensions, I should have removed that before commit. Sorry for that, Thanks!